.ci/manywheel/build_common.sh

```diff
@@ -209,12 +203,6 @@ if [[ -n "$BUILD_PYTHONLESS" ]]; then
 
     mkdir -p /tmp/$LIBTORCH_HOUSE_DIR
 
-    if [[ "$DESIRED_DEVTOOLSET" == *"cxx11-abi"* ]]; then
-        LIBTORCH_ABI="cxx11-abi-"
```

Comment:
I think we would need to set: ```LIBTORCH_ABI="cxx11-abi-```

Reply:
Why do we still need this? If all libtorch builds are now cxx11-abi is it important to still have the distinction in the filename?